src/types/shared.ts

```diff
@@ -33,6 +34,11 @@ export interface FluxToolbarFunction {
   link: string
 }
 
+// FIXME: confirm if this is really FluxFuntions?
+// or also contains other docs for additional flux language features.
+export type FluxDocs = FluxdocsRouteT
+export type FluxFunction = FluxDocs
```

Comment:
Wrote the types to make it explicit.
That we are assuming that the FluxDocs are all of flux functions.